docs/SUMMARY.md

```diff
@@ -7,6 +7,11 @@
   - [HttpProvider](rpc-provider/classes/_http_index_.httpprovider.md)
   - [WsProvider](rpc-provider/classes/_ws_index_.wsprovider.md)
 
+## Substrate Interfaces
```

Comment:
Rather call this `Runtime Node Interfaces`

Reply:
If RPCs are never exposed by the runtime, should RPC be under the "Runtime Node Interfaces" heading?

Reply:
They are expose by the Node. So it is basically -

- RPC
- Chain state (runtime)
- Extrinsics (runtime)

So I would denote the latter 2 with "(runtime)" 

In that case, the heading is misleading, you are right. (Would just remove "substrate", so basically just "Interfaces" works here as well)